tier2_standards/python_version.md

```diff
@@ -1,21 +1,19 @@
-# Software Must Use Python>=3.6 Where Applicable
+# Software Must Support Recent Version(s) of Python
 
 This is one of the tier 2 standards. See full list [on the overview page](README.md).
 
 ## Short description
 
-We require that all Python software 'support' Python version `3.6` or higher.  We define 'support' in this context such that:
+We require that all Python software 'support' one or more 'recent' versions of Python.  **Currently, we condsider 'recent' versions to include 3.6, 3.7, and 3.8.**  We define 'support' in this context such that:
```

Comment:
I saw your comment and I agree that I'm not sure where to put this. I think that if the style guide says something about python versions i'd point to that, but I don't want to make another place for people to have to go look up a style comment. So if the style guide doesn't say anything, I think keeping it in one place in this document is okay.

Reply:
I looked into what `style-guides` has and they don't mention any specific versions, they basically just say "please use Python 3":

> Python 3 is the recommended version of Python, and when possible, we strongly recommend developing in the latest version of Python. For more detail, see the software versioning and dependencies guide.

So I guess we will just keep what we have in the `python_version.md` document.

I did discover that `style-guides` links to some useful documentation about Python 3 version (https://python-3-for-scientists.readthedocs.io/en/latest/python3_transition_guide.html and http://python3porting.com/), so I will add those to the 'Useful Links' section!